### PR TITLE
Oe/86bz12uch/login stepper v3

### PR DESCRIPTION
**Pull request by guardian-001**
This PR presents the third version of the login stepper for suppliers
## Note !! : 
- I ran this PR just to ensure that the number of files changed is not high. 
- The work is not complete
- folders and files structure will be updated in the last pr
- i removed the dark mode noise in the app

### CODE REVIEW

File: src/modules/supplier/login/check-mail-banner.tsx [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 export default function CheckMailBanner() {
   return (
-    <View>
-      <HeaderTitle text="signup.headerTitle" type="custom" />
-
-      <Text>There are no account matches with this email</Text>
+    <View className="flex h-full w-full items-center justify-center gap-5">
+      <ErrorImg />
+      <Text className="text-center text-xl">
+        There are no account matches with this email
```
Comment (MohamedMegdich1) [inline src/modules/supplier/login/check-mail-banner.tsx:11]: translation

File: src/modules/supplier/login/shared/hooks/use-login-email-supplier.ts [hunk 2]
```diff
@@ -29,13 +30,19 @@ export const useLoginEmailSupplier = () => {
...
-          router.push(`(${space})/(public)/check-mail-banner`);
-        } else {
-          setError(axiosError.message);
-        }
+    console.log(formData);
+    console.log(data);
+    console.log({ email: formData.email });
+    emailCheck.mutate(data, {
+      onError: (error) => {
+        console.log(error);
```
Comment (MohamedMegdich1) [inline src/modules/supplier/login/shared/hooks/use-login-email-supplier.ts:38]: log

Comment (MohamedMegdich1) [reply src/modules/supplier/login/shared/hooks/use-login-email-supplier.ts:38]: onSuccess we don't do anythink

**Review by MohamedMegdich1 — CHANGES_REQUESTED** [review]

File: src/modules/supplier/login/check-mail-banner.tsx (hunk 1 above)
Comment (guardian-001) [reply src/modules/supplier/login/check-mail-banner.tsx:11]: fixed

File: src/modules/supplier/login/shared/hooks/use-login-email-supplier.ts (hunk 2 above)
Comment (guardian-001) [reply src/modules/supplier/login/shared/hooks/use-login-email-supplier.ts:38]: image
onsuccess just under on Error

Comment (guardian-001) [reply src/modules/supplier/login/shared/hooks/use-login-email-supplier.ts:38]: image
fixed

File: src/modules/supplier/login/check-mail-banner.tsx (hunk 1 above)
Comment (guardian-001) [reply src/modules/supplier/login/check-mail-banner.tsx]: image

**Pull request merged**